<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WalletDialog</class>
 <widget class="QDialog" name="WalletDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>241</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Wallet</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="balanceLable">
      <property name="text">
       <string>balanceLable</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="amountEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="refreshCaptcha">
        <property name="focusPolicy">
         <enum>Qt::FocusPolicy::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::DocumentRevert"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="captchaLable">
        <property name="text">
         <string>captchaLable</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="captchaEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="reciveButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>recive</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
